templates/about.html

```diff
@@ -18,6 +18,36 @@ <h2>
 		</p>
 	</section>
 
+	<section id="contribute" class="container">
+		<h2>Contribute</h2>
+
+		<p>
+			The HTTP Archive is an open source project maintained by a core group of developers and contributors from the community. Here's how you can become more involved with the project:
+		</p>
+
+		<ul class="bulleted">
+			<li>
+				<a href="https://bit.ly/http-archive-slack">Join the HTTP Archive Slack channel</a> to discuss the project with other members of the web community. This channel is also a sounding board to get ideas and feedback on new features.
+			</li>
+
+			<li>
+				<a href="https://discuss.httparchive.org">Post on the HTTP Archive forum</a> to share interesting analyses, get SQL help, and follow official product announcements.
+			</li>
+
+			<li>
+				<a href="https://github.com/HTTPArchive/httparchive.org">Contribute to the HTTP Archive on GitHub</a> to file a bug or feature request, submit a code change, or learn more about how it works under the hood.
+			</li>
+
+			<li>
+				<a href="https://docs.google.com/document/d/1D4k77yFVbk1e-l28sTC2fniqYQIyyt49OLPOuXTgx8g/edit?usp=sharing">Join the monthly maintenance Hangout</a> to get an update on administrative tasks. This meeting is open to the community and notes are made available.
```

Comment:
Aside: this hangout link triggers a "ask to join prompt", which means creator must explicitly accept the request (and be on the call). Is there a way we could have it open by default?

Reply:
Hmm I looked into this and found https://support.google.com/a/answer/6138421?hl=en&ref_topic=6052417 but it seems a hangout within an organization that is open to the public must have the "knock to enter" function :-/.. I did set that config to enable all HA org hangouts to be public by default.

Reply:
What if we created a hangout link from one of our personal accounts? Would that remove the need for knock to enter? Or, are there other gotchas associated with that route?

Reply:
Per https://support.google.com/meet/answer/7308559?hl=en "You need a G Suite account to create meetings" 😞

But this limitation seems to apply to Meet conferencing, not legacy Hangouts. So if I create a personal Calendar event with a Hangout attached, I can join from other accounts via permalink without knocking.

Updated the VC link in the doc. Let's see how well it works next time we meet 😄

Reply:
/cc @yoavweiss fyi. we have the same challenge with the webperf WG meetup, where we need to manually accept folks from a chromium.org account. Might be worth exploring similar solution there..